README.md

```diff
@@ -52,32 +54,37 @@ caching do |config|
   config.cache_client =  Dalli::Client.new(...)
   config.cache_logger = Logger.new(STDOUT) # or Rails.logger, for example
 end
-```
-
-Provides client-side response caching of service requests.  Responses are cached in memcached (using the provided cache client) in either a *last-resort* or *first-resort* manner.
-
-*last-resort* means that the cached value is only used when the service client request fails (with a
-`ServiceError`).  When using last-resort caching, a `refresh_probability` can be provided that causes the cached value
-to be refreshed probabilistically (rather than on every request).
 
-*first-resort* means that the cached value is always used, if present.  Requests to the service are only made
-when the cached value is close to expiry.
+# An example of a cached request
+multi = Px::Service::Client::Multiplexer.new
+method = :get
 
-An example of a cached request:
-
-```ruby
 req = subject.make_request(method, url)
-result = subject.cache_request(url) do
+result = subject.cache_request(url,:last_resort, refresh_probability: 1) do
   resp = nil
   multi.context do
```

Comment:
This example is wrong.  The entire call to cache request must be inside the multiplexer.